packages/clerk-js/src/ui/utils/test/createFixtures.tsx

```diff
@@ -84,23 +85,25 @@ const unboundCreateFixtures = <N extends UnpackContext<typeof ComponentContext>[
     const MockClerkProvider = (props: any) => {
       const { children } = props;
       return (
-        <CoreClerkContextWrapper clerk={clerkMock}>
-          <EnvironmentProvider value={environmentMock}>
-            <OptionsProvider value={optionsMock}>
-              <RouteContext.Provider value={routerMock}>
-                <AppearanceProvider appearanceKey={'signIn'}>
-                  <FlowMetadataProvider flow={componentName as any}>
-                    <InternalThemeProvider>
-                      <ComponentContext.Provider value={{ ...componentContextProps, componentName }}>
-                        {children}
-                      </ComponentContext.Provider>
-                    </InternalThemeProvider>
-                  </FlowMetadataProvider>
-                </AppearanceProvider>
-              </RouteContext.Provider>
-            </OptionsProvider>
-          </EnvironmentProvider>
-        </CoreClerkContextWrapper>
+        <SWRConfig value={{ provider: () => new Map(), dedupingInterval: 0 }}>
```

Comment:
We were forced to wrap our test fixture with a SWR config, as swr uses a global cache for tests and there is no easy way to clear it before every test. This lead to use the same initial response when mocking a method `organizations.getMemberships()` in our case and every test expect the first one to fail
[Related issue](https://github.com/vercel/swr/issues/781)

Reply:
💯 All tests seem to pass correctly, so it is good news this does not mess with anything else.